src/Faithlife.Utility/ArrayUtility.cs

```diff
@@ -38,7 +38,7 @@ public static bool AreEqual<T>(T[]? array1, T[]? array2, IEqualityComparer<T> co
 				return false;
 			if (array1.Length != array2.Length)
 				return false;
-			for (int nIndex = 0; nIndex < array1.Length; nIndex++)
+			for (var nIndex = 0; nIndex < array1.Length; nIndex++)
```

Comment:
Nit: Hungarian

:trollface:

Reply:
Exercise for the reader: write a Hungarian-detection analyzer with a fixer.